### PR TITLE
[13.0][mig] account invoice tax required

### PR DESCRIPTION
**Pull request by captivea-dcr**
superseeds #605 as it seems inactive

### CODE REVIEW

**Comment by captivea-dcr** [comment]
@kongrattapong suggestions you made in #605 are done here. BTW, do you know why travis didn't run on this PR?

**Review by victoralmau — APPROVED** [review]

**Comment by captivea-dcr** [comment]
@pedrobaeza any idea why travis did not run on this PR?

**Comment by pedrobaeza** [comment]
Yesterday status: http://status.github.com/

File: account_invoice_tax_required/tests/test_account_move_tax_required.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                partner_id=self.partner.id,
+                account_id=self.account_rec1_id.id,
+                invoice_line_ids=invoice_line_data,
+            )
+        )
+
+    def test_exception(self):
+        """Validate invoice without tax must raise exception
+        """
+        with self.assertRaises(exceptions.Warning):
```
Comment (bealdav) [inline account_invoice_tax_required/tests/test_account_move_tax_required.py]: Warning is deprecated, please use an appropriate one

**Review by jbaudoux — APPROVED** [review]
LGTM
README must be updated (see https://github.com/OCA/maintainer-tools)

**Comment by captivea-dcr** [comment]
@bealdav @pedrobaeza is this one ok? Thanks for your time

**Comment by pedrobaeza** [comment]
At least all migration commits should be squashed

**Comment by captivea-dcr** [comment]
@pedrobaeza @bealdav @kongrattapong is this one ok for you? can we merge it? thanks

**Comment by kittiu** [comment]
@captivea-dcr almose there, but can you do us a favor 
1. To have 2 commit as suggested here https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#technical-method-to-migrate-a-module-from-120-to-130-branch
2. Can you cover more test?

**Comment by etobella** [comment]
@captivea-dcr Any news?

**Comment by albariera** [comment]
This PR has been superseded as it seems inactive

**Pull request closed without merging**